### PR TITLE
update comfui_stablediffusion, add checkpoints and lora

### PR DESCRIPTION
**Pull request by pschroedl**
Performance updates to ComfyUI_StableDiffusion from:
https://github.com/pschroedl/ComfyUI-StreamDiffusion/pull/6
Example Workflow Updates:
https://github.com/pschroedl/ComfyUI-StreamDiffusion/pull/7

Node now loads safetensor files ( full huggingface checkpoint loading is broken ), this PR adds checkpoints for:
sd_turbo.safetensors, kohaku_v2.1.safetensors and
https://huggingface.co/chenxxiao/3dCartoonVision_v10

and lora : ral-dissolve1.5
https://civitai.com/models/245889?modelVersionId=314246

### CODE REVIEW

File: runner/dl_checkpoints.sh [hunk 1]
```diff
@@ -106,6 +107,17 @@ function download_sam2_checkpoints() {
 function download_stable_diffusion_checkpoints() {
     huggingface-cli download KBlueLeaf/kohaku-v2.1 --local-dir models/ComfyUI--models/checkpoints --include "*.safetensors"
     huggingface-cli download stabilityai/sd-turbo --local-dir models/ComfyUI--models/checkpoints --include "*.safetensors"
+
+    # ComfyUI_StreamDIffusion is loading single file safetensors from /models/checkpoints
+    wget -P models/checkpoints/ https://huggingface.co/KBlueLeaf/kohaku-v2.1/resolve/main/kohaku-v2.1.safetensors
+    wget -P models/checkpoints/ https://huggingface.co/stabilityai/sd-turbo/resolve/main/sd_turbo.safetensors
+    # chenxxiao/3dCartoonVision_v10
+    wget -P models/checkpoints https://huggingface.co/chenxxiao/3dCartoonVision_v10/resolve/main/3dCartoonVision_v10.safetensors?download=true --content-disposition
```
Comment (leszko) [inline runner/dl_checkpoints.sh:115]:
```suggestion
    huggingface-cli download KBlueLeaf/kohaku-v2.1 --local-dir models/checkpoints --include "*.safetensors"
    huggingface-cli download stabilityai/sd-turbo --local-dir models/checkpoints --include "*.safetensors"

    # chenxxiao/3dCartoonVision_v10
    huggingface-cli download chenxxiao/3dCartoonVision_v10 --local-dir models/checkpoints --include "*.safetensors"
```

1. For the sake of consistency, can we change everything to get downloaded with `huggingface-cli`? This what I suggested worked for me
2. Do I understand correctly that the streamdiffusion node takes models from `/models/checkpoints` and not from the comfyui workspace directory `/comfui/models`?

Comment (pschroedl) [reply runner/dl_checkpoints.sh]: Created HF-hub repos for checkpoints and Loras to get rid of wget downloads.

The streamdiffusion node takes models the comfyui workspace directory `/comfui/models/checkpoints`.

File: runner/dl_checkpoints.sh [hunk 2]
```diff
@@ -106,6 +107,17 @@ function download_sam2_checkpoints() {
...
+}
+
+function download_stable_diffusion_loras() {
+    # ral-dissolve-sd15 LoRA
+    wget -P models/loras https://civitai.com/api/download/models/314246?type=Model&format=SafeTensor --content-disposition
```
[the remark this answers is not in the document]
Comment (pschroedl) [reply runner/dl_checkpoints.sh]: moved to HF

**Comment by pschroedl** [comment]
> Could you point me to the API JSON workflow that I can test?

All should be updated now to use the correctly named nodes.
The primary ones to test are:
* img2img_no_acceleration_turbo_API_FORMAT.json  
-> just sd_turbo_15
* img2img_no_acceleration_w_lora_API_FORMAT.json  
-> 3dCartoonVision + ral-dissolve lora

I've removed the older, outdated workflows from the repo

File: runner/dl_checkpoints.sh [hunk 3]
```diff
@@ -104,9 +105,14 @@ function download_sam2_checkpoints() {
...
-    huggingface-cli download KBlueLeaf/kohaku-v2.1 --local-dir models/ComfyUI--models/checkpoints --include "*.safetensors"
-    huggingface-cli download stabilityai/sd-turbo --local-dir models/ComfyUI--models/checkpoints --include "*.safetensors"
+function download_stream_diffusion_checkpoints() {
+    # ComfyUI_StreamDiffusion is loading single file safetensors from /models/checkpoints
+    huggingface-cli download pschroedl/comfystream_checkpoints --local-dir models/checkpoints --include "*.safetensors"
+}
+
+function download_stream_diffusion_loras() {
+    # ral-dissolve-sd15 LoRA
+    huggingface-cli download pschroedl/comfystream_loras --local-dir models/loras --include "*.safetensors"
```
Comment (leszko) [inline runner/dl_checkpoints.sh:115]:
```suggestion
function download_stream_diffusion_checkpoints() {
    # ComfyUI_StreamDiffusion is loading single file safetensors from /models/checkpoints
    huggingface-cli download pschroedl/comfystream_checkpoints --local-dir models/ComfyUI--models/checkpoints --include "*.safetensors"
}

function download_stream_diffusion_loras() {
    # ral-dissolve-sd15 LoRA
    huggingface-cli download pschroedl/comfystream_loras --local-dir models/ComfyUI--models/loras --include "*.safetensors"
```

**Pull request merged**